CHANGELOG.md

```diff
@@ -23,6 +23,12 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
 
 - Added Makefile to simplify building and developing the project locally. (#67)
 
+- Added test results failed/skipped/passed numbers to each build in a project's
+  build list. (#53)
+
+- Added test result details, showing test messages and per-artifact test
+  results, to the build details page. (#53)
+
```

Comment:
v1.5.0 release has been merged (https://github.com/iver-wharf/wharf-web/pull/86). Suggest moving this to a new WIP release of v1.6.0